pkg/rtc/participant.go

```diff
@@ -1475,6 +1476,10 @@ func (p *ParticipantImpl) onStreamStateChange(update *sfu.StreamStateUpdate) err
 }
 
 func (p *ParticipantImpl) onSubscribedMaxQualityChange(trackID livekit.TrackID, subscribedQualities []*livekit.SubscribedCodec, maxSubscribedQualites []types.SubscribedCodecQuality) error {
+	if p.params.DisableDynacast {
+		return nil
+	}
```

Comment:
Note that this disables the dynacast path completely. There is some code 👇 which sends analytics events about max layer. Do we want to send those still, but just not send the message back to the client?

Reply:
Does that means the dashboard will not show subscribed layers? If it is, I think that is needed

Reply:
Yes @cnderrauber. Can change it so that telemetry events are sent, but messages to the client are throttled.
But, that means dashboard will show subscribed layer max requirements, but it will not show max published layer changing. Guess, that is okay. wdyt? @davidzhao wdyt?

Reply:
I think it's ok to skip dashboard events. They should be in sync with what's actually happening.